CHANGELOG.next.asciidoc

```diff
@@ -58,7 +58,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 
 *Filebeat*
-- [Auditbeat System Package] Added support for Apple Silicon chips. {pull}34433[34433] 
+- [Auditbeat System Package] Added support for Apple Silicon chips. {pull}34433[34433]
```

Comment:
nit: is this change needed? If not please revert.

Reply:
I think they were automatic. But what is the problem in removing one space?

Reply:
In principle a PR should not introduce changes that are not intentional, even if it is just one space or an empty line.
For this one I think that the editor is correct and removes the space that is not needed, so you can leave it as if you prefer since it is an improvement :).